wordless/theme_builder/vanilla_theme/webpack.env.js

```diff
@@ -5,9 +5,7 @@ devOptions = {
   styleLoaders: [
     {
       loader: MiniCssExtractPlugin.loader,
-      options: {
-        publicPath: '',
-      }
+      options: {}
```

Comment:
Probably I'd opt to remove the empty options. Here and in line 32.

Reply:
I've done a refactor in https://github.com/welaika/wordless/pull/261/commits/6b95f16c66fc6541d5a955a20bcedf7abee63dc0

Reply:
hell yeah